pkg/registry/common/refresh/nse_registry_client.go

```diff
@@ -31,15 +31,15 @@ import (
 )
 
 type refreshNSEClient struct {
-	client                    registry.NetworkServiceEndpointRegistryClient
-	nsesMutex                 sync.Mutex
-	nseCancels                map[string]context.CancelFunc
-	retryDelay                time.Duration
-	defaultExpirationDuration time.Duration
+	client                registry.NetworkServiceEndpointRegistryClient
+	nsesMutex             sync.Mutex
+	nseCancels            map[string]context.CancelFunc
+	retryDelay            time.Duration
+	defaultExpiryDuration time.Duration
 }
 
-func (c *refreshNSEClient) setDefaultExpiration(duration time.Duration) {
-	c.defaultExpirationDuration = duration
+func (c *refreshNSEClient) setDefaultExpiryDuration(duration time.Duration) {
+	c.defaultExpiryDuration = duration
```

Comment:
Why make this a function rather than doing it inline?

Reply:
You are right we can just remove this inline function )